### DISCUSSION TITLE
--enable-gpl configure option

### DISCUSSION THREAD

**Discussion by leslie-wang** [libvips/libvips#2027] [Ideas]
I know most libraries used by libvips are LGPL or BSD, but probably some libraries are GPL. So is it possible to have one configuration option `--enable-gpl` or `--disable-gpl` to automatically disable GPLed libraries. I borrowed the idea from FFMPEG / AVCONV.

**Comment by jcupitt** [comment]
That's a good idea. Let's tag this as an enhancement.
